featurebyte/query_graph/sql/tile_compute.py

```diff
@@ -80,22 +79,14 @@ def construct_tile_sqls(self) -> dict[str, Select]:
         for tile_info in self.tile_infos:
             # Convert template SQL with concrete start and end timestamps, based on the requested
             # point-in-time and feature window sizes
-            tile_sql_with_start_end_expr = get_tile_sql_from_point_in_time(
+            tile_sql_expr = get_tile_sql_from_point_in_time(
                 sql_template=tile_info.sql_template,
                 point_in_time_list=self.point_in_time_list,
                 frequency=tile_info.frequency,
                 time_modulo_frequency=tile_info.time_modulo_frequency,
                 blind_spot=tile_info.blind_spot,
                 window=self.get_max_window_size(tile_info.tile_table_id),
             )
-            # Include global tile index that would have been computed by F_TIMESTAMP_TO_INDEX UDF
-            # during scheduled tile jobs
```

Comment:
The query above already contains the global tile index, so the following is no longer needed.